_layouts/partners/sba-impact-story.html

```diff
@@ -10,8 +10,8 @@
     <section>
         <div class="container partners-container">
             <div class="grid-row">
+                <h1 class="partners-title">{{ page.title }}</h1>
```

Comment:
I would expect `grid-row` to only contain grid cell elements, which probably explains why it was necessary to add the `width: 100%` style.

Maybe we can add a separate row for the header, or remove it from the grid altogether?

```suggestion
            <h1 class="partners-title">{{ page.title }}</h1>
            <div class="grid-row">
```

Reply:
Agreed, its cleaner to get it out of the grid. I moved it out in the faq and sba files